src/styles/variables.sass

```diff
@@ -0,0 +1,49 @@
+// Favorite constants
+$gratio: 1.618
+$pi: 3.14159
+
+// Dimensions
+$radius: 2px
+$spacing: 24px
+$padding: 10px
+
+// Font properties
+$font-family-sans-serif: 'Helvetica Neue', Helvetica, Arial, sans-serif
+$font-family-serif: Georgia, Cambria, 'Times New Roman', Times, serif
+$font-family-monospace: Monaco, Menlo, Consolas, 'Courier New', monospace
+
+$font-family-base: $font-family-sans-serif
+$font-family-head: $font-family-serif
+
+$base-font-size: 16px
+$fw-normal: 400
+$fw-medium: 500
+$fw-semibold: 600
+$fw-bold: 700
```

Comment:
A co až budeš chtít použít light, semi-bold nebo extra-bold? To budeme mixovat `bold` a `300`? Pak si ta čísla člověk stejně nakonec pamatovat musí, nemá-li proměnné.


> na Darujme v intervalu jednoho měsíce používáš

@Onset Tak jsi mi to měl omlátit o hlavu při review. To, že nejsem neomylný, tady přece vůbec relevantní není…